ui/desktop/app/routes/scopes/scope/projects/targets/target.js

```diff
@@ -65,6 +70,22 @@ export default class ScopesScopeProjectsTargetsTargetRoute extends Route {
     }
   }
 
+  /**
+   * Sets the 'isConnecting' queryParam to false if connection failed.
+   * @returns {boolean}
+   */
+  @action
+  didTransition() {
+    if (this.connectionError) {
```

Comment:
Should we set the connection error back to false here after we change the query param value? Otherwise it will always be true since it never gets reset